### ISSUE TITLE
Support speak on demand

### ISSUE THREAD

**Issue by josephsl** [josephsl/enhancedTouchGestures#3]
Hi,

Let us support speak on demand mode (NVDA 2024.1). We can try the dictionary method to make it compatible with 2023.3.4 and earlier, or go all the way in and use the speak on demand flag right away (2024.1 and later). Of these, I vote for the second option.

Thanks.

**Comment by kefaslungu** [comment]
hi,

I've began working on it.

the only issue is: I'll like to add speak on demand feature to the synth settings too, but if nvda is on speak on demand mode, switching touch mode is not announced. And the user has to know when they are in synth settings mode to change the settings.

**Comment by kefaslungu** [comment]
hi,

I also don't want to support nvda earlier than 2024.1
